packages/sources/alongside-test/src/endpoint/router.ts

```diff
@@ -0,0 +1,30 @@
+import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/meta'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { customSettings } from '../config'
+import { ProviderResponseBody, httpTransport, inputParameters, RequestParams } from './balances'
+
+export type EndpointTypes = {
+  Request: {
+    Params: RequestParams
+  }
+  Response: SingleNumberResultResponse
+  CustomSettings: typeof customSettings
+  Provider: {
+    RequestBody: never
+    ResponseBody: ProviderResponseBody
+  }
+}
+
+export const routingTransport = new RoutingTransport<EndpointTypes>(
+  {
+    HTTP: httpTransport,
+  },
+  () => 'HTTP',
+)
```

Comment:
Since there's only one transport for this endpoint, we can skip on using a routing transport. Instead you can pass the HTTP transport directly to the endpoint below.